src/components/RemoteBrowserDialog.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+    <form @submit.prevent="submit">
+        <div ref="modal" class="modal fade" tabindex="-1" data-bs-backdrop="static">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 id="exampleModalLabel" class="modal-title">
+                            {{ $t("Add a Remote Browser") }}
+                        </h5>
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close" />
+                    </div>
+                    <div class="modal-body">
+                        <div class="mb-3">
+                            <label for="remote-browser-name" class="form-label">{{ $t("Friendly Name") }}</label>
+                            <input id="remote-browser-name" v-model="remoteBrowser.name" type="text" class="form-control" required>
+                        </div>
+
+                        <div class="mb-3">
+                            <label for="remote-browser-url" class="form-label">{{ $t("URL") }}</label>
+                            <input id="remote-browser-url" v-model="remoteBrowser.url" type="text" class="form-control" required>
+
+                            <div class="form-text mt-3">
+                                {{ $t("Examples") }}:
+                                <ul>
+                                    <li>wss://chrome.browserless.io/playwright?token=YOUR-API-TOKEN</li>
```

Comment:
`ws://` should be better. I think non-developers may not be faimilar with the WebSocket protocol and I think people will likely to test it without TLS first.